2024-05-26 16:22:56,299 fail2ban.server         [400]: INFO    --------------------------------------------------
2024-05-26 16:22:56,299 fail2ban.server         [400]: INFO    Starting Fail2ban v0.11.2
2024-05-26 16:22:56,300 fail2ban.observer       [400]: INFO    Observer start...
2024-05-26 16:22:56,399 fail2ban.database       [400]: INFO    Connected to fail2ban persistent database '/var/lib/fail2ban/fail2ban.sqlite3'
2024-05-26 16:22:56,401 fail2ban.jail           [400]: INFO    Creating new jail 'sshd'
2024-05-26 16:22:56,413 fail2ban.jail           [400]: INFO    Jail 'sshd' uses pyinotify {}
2024-05-26 16:22:56,417 fail2ban.jail           [400]: INFO    Initiated 'pyinotify' backend
2024-05-26 16:22:56,419 fail2ban.filter         [400]: INFO      maxLines: 1
2024-05-26 16:22:56,450 fail2ban.filter         [400]: INFO      maxRetry: 3
2024-05-26 16:22:56,450 fail2ban.filter         [400]: INFO      findtime: 600
2024-05-26 16:22:56,451 fail2ban.actions        [400]: INFO      banTime: 7200
2024-05-26 16:22:56,451 fail2ban.filter         [400]: INFO      encoding: UTF-8
2024-05-26 16:22:56,452 fail2ban.filter         [400]: INFO    Added logfile: '/var/log/auth.log' (pos = 867, hash = 0092a772cd78cdfee51306f00351af8ac4ca52a1)
2024-05-26 16:22:56,457 fail2ban.jail           [400]: INFO    Jail 'sshd' started
2024-05-26 16:55:58,714 fail2ban.filter         [400]: INFO    [sshd] Found 24.86.24.167 - 2024-05-26 16:55:58
2024-05-26 16:56:00,654 fail2ban.filter         [400]: INFO    [sshd] Found 24.86.24.167 - 2024-05-26 16:56:00
2024-05-26 16:56:02,697 fail2ban.filter         [400]: INFO    [sshd] Found 24.86.24.167 - 2024-05-26 16:56:02
2024-05-26 16:56:03,354 fail2ban.actions        [400]: NOTICE  [sshd] Ban 24.86.24.167
2024-05-26 17:22:09,224 fail2ban.server         [400]: INFO    Shutdown in progress...
2024-05-26 17:22:09,225 fail2ban.observer       [400]: INFO    Observer stop ... try to end queue 5 seconds
2024-05-26 17:22:09,246 fail2ban.observer       [400]: INFO    Observer stopped, 0 events remaining.
2024-05-26 17:22:09,289 fail2ban.server         [400]: INFO    Stopping all jails
2024-05-26 17:22:09,289 fail2ban.filter         [400]: INFO    Removed logfile: '/var/log/auth.log'
2024-05-26 17:22:09,551 fail2ban.actions        [400]: NOTICE  [sshd] Flush ticket(s) with iptables-multiport
2024-05-26 17:22:09,569 fail2ban.actions        [400]: NOTICE  [sshd] Unban 24.86.24.167
2024-05-26 17:22:09,603 fail2ban.jail           [400]: INFO    Jail 'sshd' stopped
2024-05-26 17:22:09,603 fail2ban.database       [400]: INFO    Connection to database closed.
2024-05-26 17:22:09,604 fail2ban.server         [400]: INFO    Exiting Fail2ban
2024-05-26 19:07:54,854 fail2ban.server         [393]: INFO    --------------------------------------------------
2024-05-26 19:07:54,854 fail2ban.server         [393]: INFO    Starting Fail2ban v0.11.2
2024-05-26 19:07:54,855 fail2ban.observer       [393]: INFO    Observer start...
2024-05-26 19:07:55,011 fail2ban.database       [393]: INFO    Connected to fail2ban persistent database '/var/lib/fail2ban/fail2ban.sqlite3'
2024-05-26 19:07:55,013 fail2ban.jail           [393]: INFO    Creating new jail 'sshd'
2024-05-26 19:07:55,023 fail2ban.jail           [393]: INFO    Jail 'sshd' uses pyinotify {}
2024-05-26 19:07:55,026 fail2ban.jail           [393]: INFO    Initiated 'pyinotify' backend
2024-05-26 19:07:55,027 fail2ban.filter         [393]: INFO      maxLines: 1
2024-05-26 19:07:55,263 fail2ban.filter         [393]: INFO      maxRetry: 3
2024-05-26 19:07:55,263 fail2ban.filter         [393]: INFO      findtime: 600
2024-05-26 19:07:55,263 fail2ban.actions        [393]: INFO      banTime: 7200
2024-05-26 19:07:55,264 fail2ban.filter         [393]: INFO      encoding: UTF-8
2024-05-26 19:07:55,264 fail2ban.filter         [393]: INFO    Added logfile: '/var/log/auth.log' (pos = 1782, hash = e4fc0c608dadb2285474b5fcb60103d5623df864)
2024-05-26 19:07:55,333 fail2ban.jail           [393]: INFO    Jail 'sshd' started
2024-05-26 20:07:06,583 fail2ban.server         [393]: INFO    Shutdown in progress...
2024-05-26 20:07:06,583 fail2ban.observer       [393]: INFO    Observer stop ... try to end queue 5 seconds
2024-05-26 20:07:06,604 fail2ban.observer       [393]: INFO    Observer stopped, 0 events remaining.
2024-05-26 20:07:06,644 fail2ban.server         [393]: INFO    Stopping all jails
2024-05-26 20:07:06,645 fail2ban.filter         [393]: INFO    Removed logfile: '/var/log/auth.log'
2024-05-26 20:07:06,770 fail2ban.actions        [393]: NOTICE  [sshd] Flush ticket(s) with iptables-multiport
2024-05-26 20:07:07,849 fail2ban.jail           [393]: INFO    Jail 'sshd' stopped
2024-05-26 20:07:07,849 fail2ban.database       [393]: INFO    Connection to database closed.
2024-05-26 20:07:07,855 fail2ban.server         [393]: INFO    Exiting Fail2ban
2024-05-27 18:28:08,954 fail2ban.server         [396]: INFO    --------------------------------------------------
2024-05-27 18:28:08,955 fail2ban.server         [396]: INFO    Starting Fail2ban v0.11.2
2024-05-27 18:28:08,955 fail2ban.observer       [396]: INFO    Observer start...
2024-05-27 18:28:09,065 fail2ban.database       [396]: INFO    Connected to fail2ban persistent database '/var/lib/fail2ban/fail2ban.sqlite3'
2024-05-27 18:28:09,067 fail2ban.jail           [396]: INFO    Creating new jail 'sshd'
2024-05-27 18:28:09,077 fail2ban.jail           [396]: INFO    Jail 'sshd' uses pyinotify {}
2024-05-27 18:28:09,080 fail2ban.jail           [396]: INFO    Initiated 'pyinotify' backend
2024-05-27 18:28:09,081 fail2ban.filter         [396]: INFO      maxLines: 1
2024-05-27 18:28:09,104 fail2ban.filter         [396]: INFO      maxRetry: 3
2024-05-27 18:28:09,104 fail2ban.filter         [396]: INFO      findtime: 600
2024-05-27 18:28:09,105 fail2ban.actions        [396]: INFO      banTime: 7200
2024-05-27 18:28:09,105 fail2ban.filter         [396]: INFO      encoding: UTF-8
2024-05-27 18:28:09,106 fail2ban.filter         [396]: INFO    Added logfile: '/var/log/auth.log' (pos = 11199, hash = e4fc0c608dadb2285474b5fcb60103d5623df864)
2024-05-27 18:28:09,111 fail2ban.jail           [396]: INFO    Jail 'sshd' started
2024-05-27 19:27:00,815 fail2ban.server         [396]: INFO    Shutdown in progress...
2024-05-27 19:27:00,815 fail2ban.observer       [396]: INFO    Observer stop ... try to end queue 5 seconds
2024-05-27 19:27:00,844 fail2ban.observer       [396]: INFO    Observer stopped, 0 events remaining.
2024-05-27 19:27:00,876 fail2ban.server         [396]: INFO    Stopping all jails
2024-05-27 19:27:00,876 fail2ban.filter         [396]: INFO    Removed logfile: '/var/log/auth.log'
2024-05-27 19:27:01,653 fail2ban.actions        [396]: NOTICE  [sshd] Flush ticket(s) with iptables-multiport
2024-05-27 19:27:02,079 fail2ban.jail           [396]: INFO    Jail 'sshd' stopped
2024-05-27 19:27:02,079 fail2ban.database       [396]: INFO    Connection to database closed.
2024-05-27 19:27:02,080 fail2ban.server         [396]: INFO    Exiting Fail2ban
2024-05-28 11:04:21,369 fail2ban.server         [393]: INFO    --------------------------------------------------
2024-05-28 11:04:21,443 fail2ban.server         [393]: INFO    Starting Fail2ban v0.11.2
2024-05-28 11:04:21,445 fail2ban.observer       [393]: INFO    Observer start...
2024-05-28 11:04:21,533 fail2ban.database       [393]: INFO    Connected to fail2ban persistent database '/var/lib/fail2ban/fail2ban.sqlite3'
2024-05-28 11:04:21,534 fail2ban.jail           [393]: INFO    Creating new jail 'sshd'
2024-05-28 11:04:21,544 fail2ban.jail           [393]: INFO    Jail 'sshd' uses pyinotify {}
2024-05-28 11:04:21,547 fail2ban.jail           [393]: INFO    Initiated 'pyinotify' backend
2024-05-28 11:04:21,549 fail2ban.filter         [393]: INFO      maxLines: 1
2024-05-28 11:04:21,579 fail2ban.filter         [393]: INFO      maxRetry: 3
2024-05-28 11:04:21,580 fail2ban.filter         [393]: INFO      findtime: 600
2024-05-28 11:04:21,580 fail2ban.actions        [393]: INFO      banTime: 7200
2024-05-28 11:04:21,580 fail2ban.filter         [393]: INFO      encoding: UTF-8
2024-05-28 11:04:21,581 fail2ban.filter         [393]: INFO    Added logfile: '/var/log/auth.log' (pos = 12814, hash = e4fc0c608dadb2285474b5fcb60103d5623df864)
2024-05-28 11:04:21,585 fail2ban.jail           [393]: INFO    Jail 'sshd' started
2024-05-28 11:05:00,693 fail2ban.filter         [393]: INFO    [sshd] Found 79.95.86.16 - 2024-05-28 11:05:00
2024-05-28 11:05:06,074 fail2ban.filter         [393]: INFO    [sshd] Found 79.95.86.16 - 2024-05-28 11:05:06
2024-05-28 11:05:15,625 fail2ban.filter         [393]: INFO    [sshd] Found 79.95.86.16 - 2024-05-28 11:05:15
2024-05-28 11:05:15,662 fail2ban.actions        [393]: NOTICE  [sshd] Ban 79.95.86.16
2024-05-28 11:29:14,614 fail2ban.actions        [393]: NOTICE  [sshd] Unban 79.95.86.16
2024-05-28 11:29:30,924 fail2ban.server         [393]: INFO    Shutdown in progress...
2024-05-28 11:29:30,925 fail2ban.observer       [393]: INFO    Observer stop ... try to end queue 5 seconds
2024-05-28 11:29:30,945 fail2ban.observer       [393]: INFO    Observer stopped, 0 events remaining.
2024-05-28 11:29:30,986 fail2ban.server         [393]: INFO    Stopping all jails
2024-05-28 11:29:30,986 fail2ban.filter         [393]: INFO    Removed logfile: '/var/log/auth.log'
2024-05-28 11:29:31,541 fail2ban.actions        [393]: NOTICE  [sshd] Flush ticket(s) with iptables-multiport
2024-05-28 11:29:31,590 fail2ban.jail           [393]: INFO    Jail 'sshd' stopped
2024-05-28 11:29:31,591 fail2ban.database       [393]: INFO    Connection to database closed.
2024-05-28 11:29:31,591 fail2ban.server         [393]: INFO    Exiting Fail2ban
